### ISSUE TITLE
Upgrade to latest codegen?

### ISSUE THREAD

**Issue by dotansimha** [agoda-com/graphql-codegen-csharp#31]
Hi all! 

I'm Dotan, the creator of [`graphql-code-generator`](https://graphql-code-generator.com), and part of [The Guild](https://medium.com/the-guild). 
I recently found this repo, and it looks great! Awesome idea! 
I would like to list this repo as part of the plugins list in our website, but it needs to be updated to the latest codegen API. I can help with that :)

Also,  would like to schedule a short call for talking about this plugin? how to do use it and how can we help to improve it?

**Comment by jenol** [comment]
Yes! Being listed on the plugins page would be awesome. I will prioritize the codegen upgrade. Help is welcome. We are using the code gen to create C# clients for sangria-graphql Scala services. The generated Query classes plug into our web client. I can show you what we have so far.

**Comment by dicko2** [comment]
yea sounds good, we should move it to our latest RR client though, and open that up (currently its on the enterprise instance).

I'll talk to 200 about OS the RR client and we should get the template updated to use new RR and the latest codegen API in a couple weeks after we live the current sprints work on host-cal

@dotansimha sure we can do a call as well

**Comment by chdanielmueller** [comment]
Hi there,
Are there any news about this?
Thanks,
Daniel
